nifi-nar-bundles/nifi-amqp-bundle/nifi-amqp-processors/src/main/java/org/apache/nifi/amqp/processors/AMQPPublisher.java

```diff
@@ -16,14 +16,14 @@
  */
 package org.apache.nifi.amqp.processors;
 
-import java.io.IOException;
-
-import org.apache.nifi.logging.ComponentLog;
```

Comment:
Hey @m-hogue, thanks for this PR. Can you take care of the not needed changes that have been (most likely) introduced by your IDE? For example this class is marked as changed but there is no actual change in it. It's similar for some files modified by your PR. Thanks!